pkg/session/bootstrap.go

```diff
@@ -3575,7 +3575,7 @@ func rebuildAllPartitionValueMapAndSorted(s *session) {
 			if pi == nil || pi.Type != model.PartitionTypeList {
 				continue
 			}
-			tbl, ok := is.TableByID(s.currentCtx, t.ID)
+			tbl, ok := is.TableByID(context.Background(), t.ID)
```

Comment:
Why here need update bootstrap part this line code？

Reply:
During my testing, I triggered several panic cases at this place.

Reply:
> I triggered several panic cases at this place.

how is that related to context change?